container.go

```diff
@@ -346,7 +346,7 @@ func newContainers(pod *Pod, contConfigs []ContainerConfig) ([]*Container, error
 }
 
 // createContainer creates and start a container inside a Pod.
-func createContainer(pod *Pod, contConfig ContainerConfig) (*Container, error) {
+func createContainer(pod *Pod, contConfig ContainerConfig) (VCContainer, error) {
```

Comment:
I don't think this function needs to return `VCContainer` type since it is only used from the virtcontainers implementation. Neither the mock implementation, nor any external consumer of the lib can access this function as it is private. Leaving this function unchanged simplifies the changes above (in `api.go`).